hw/ip/otbn/doc/_index.md

```diff
@@ -519,6 +519,46 @@ The `URND` LFSR is seeded once from the EDN connected via `edn_urnd` when OTBN s
 Each new execution of OTBN will reseed the `URND` LFSR.
 The LFSR state is advanced every cycle when OTBN is running.
 
+### Operational States {#design-details-operational-states}
+
+OTBN can be in one of two operational states: busy, or idle.
```

Comment:
Happy for this to be a follow-up, but I still don't think I know whether you mean 2 states ("busy" and "idle", with "locked" to follow) or whether you mean that there are several busy states (splitting out things like wiping operations), so "busy" is a property which is true of some of the states. This sentence seems to be the former but the text below talks about transitioning into "**a** busy state", which implies the latter.